README.md

```diff
@@ -219,6 +221,29 @@ replicas and the one `hello-osiris` pod should be terminated.
 Make a request again, and watch as Osiris scales the deployment back to one
 replica and your request is handled successfully.
 
+### Features
+
+#### Counting requests at the proxy level
```

Comment:
I'm not sure if this feature is sufficiently important to warrant its own section, especially given that relevant config options are already covered in the tables above.

Could we consider dropping this section?

Reply:
ok, I removed it.